dashboard/pages/fragment_graph.tsx

```diff
@@ -168,8 +169,8 @@ function buildFragmentDependencyAsEdges(
 
 const SIDEBAR_WIDTH = 200
 
-type BackPressureAlgo = "p50" | "p90" | "p95" | "p99"
-const backPressureAlgos: BackPressureAlgo[] = ["p50", "p90", "p95", "p99"]
+type BackPressureAlgo = "p50" | "p90" | "p95" | "p99" | "current"
```

Comment:
Please make `current` the default option. Previously we use `Disabled` by default because Prometheus is not always available, but now `current` don't depend on it.

Reply:
Also, it seems too obscure to users that `p50/90/99` is from Prometheus while `current` is from Meta. A better way is probably making two `ListBox`: 1) Data source: Embedded or Prometheus 2) Only shows for Prometheus: average/p50/p99